packages/db/src/translate/providers.ts

```diff
@@ -27,7 +27,7 @@ export const translatePostgresProviderToApiProvider = (
     createdAt: record.created_at.getTime(),
     updatedAt: record.updated_at.getTime(),
     username: record.username,
-    password: record.username,
+    password: record.password,
```

Comment:
I'm concerned this passed phase 2 tests.  Is there a P2 bug we're missing, or integration test coverage missing for ftp/authed providers? 

I added a test in #2870 to cover PDRs with node providers that have passwords, but surely this should have failed elsewhere?   Or are we not using API returns to get providers for the relevant tasks?   :thinking:

Reply:
More importantly this didn't require a change to the units?!

We should probably change u/p to not both be the *same* string in `translatePostgresProviderToApiProvider translates the expected API record`

Reply:
Does this need backported to p2?

Reply:
So actually we should talk about this because I originally thought this was causing some trouble for me in terms of passing ftp units.  It was totally unrelated and doesn't even need to be in this PR.  No changes to units or anything were needed after this change.  I will go change that test `translatePostgresProviderToApiProvider translates the expected API record` and see how this behaves. and follow up with more info